### ISSUE TITLE
Deleting Multivalued Sections rows on iOS 12

### ISSUE THREAD

**Issue by silviuandrica** [xmartlabs/Eureka#1651]
Hi,

When running the 4.2.0 Eureka example (`pod try Eureka`) on iOS 12, in the Multivalued examples -> Multivalued Sections example, tapping on the `-` red button at the left of a row does nothing, and the row is not deleted. Adding new rows, though, works. On iOS 11, though, deleting works.

I am using Xcode 10.0 beta 6.

Cheers,
Silviu

**Comment by oxview** [comment]
Hi @silviuandrica ,

The fastest fix is to override `FormViewController` methods:

```swift
@available(iOS 11.0, *)
override func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
    guard !self.form[indexPath].trailingSwipe.actions.isEmpty else {
        return nil
    }
    
    return super.tableView(tableView, trailingSwipeActionsConfigurationForRowAt: indexPath)
}

@available(iOS 11.0, *)
override func tableView(_ tableView: UITableView, leadingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
    guard !self.form[indexPath].leadingSwipe.actions.isEmpty else {
        return nil
    }
    
    return super.tableView(tableView, leadingSwipeActionsConfigurationForRowAt: indexPath)
}
```
Cheers 😉

**Comment by silviuandrica** [comment]
Thanks @oxview , it works like a charm, both iOS 11.4 and 12.

Cheers!
